pipelines/rj_smas__disparo_pic_lembrete/tasks.py

```
@@ -0,0 +1,266 @@
+@task
+def printar(text):
+    """exibe o texto passado como parâmetro"""
+    log(f"Printando {text}")
+
+
+@task
+def get_destinations(
+    destinations: Union[None, List[str]],
+    query: str,
+    billing_project_id: str = "rj-smas",
+    query_processor_name: str = None,
```

Ruff (0.14.1)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)